src/auth/baseexternalclient.ts

```diff
@@ -378,10 +378,11 @@ export abstract class BaseExternalAccountClient extends AuthClient {
         stsResponse.access_token
       );
     } else {
```

Comment:
If you made this `else if (stsResponse.expires_in) {`

Then there's no need to cast to a `number`.